src/main/java/org/l2x6/rpkgtests/CreateTestJarsXmlMojo.java

```diff
@@ -96,24 +95,20 @@
     @Parameter(property = "rpkgtests.encoding", defaultValue = "${project.build.sourceEncoding}")
     private String encoding;
 
+    @Parameter(property = "rpkgtests.activatingPropertyName", defaultValue = "rpkgtests.activating.property")
+    private String activatingPropertyName;
```

Comment:
Would you mind adding a sentence of JavaDoc explaining what the new parameter does? `@since 0.11.0` would be nice too.

Reply:
Added, please check if it's explanatory enough.

Reply:
Perfect, thanks!